apps/web/components/modals/search-modal.tsx

```
@@ -55,11 +55,15 @@ export function SearchModal({ serverId, onClose, onJumpToMessage }: Props) {
       <div className="w-full max-w-2xl rounded-xl overflow-hidden shadow-2xl flex flex-col" style={{ background: "#2b2d31", maxHeight: "70vh" }}>
         <div className="flex items-center gap-3 px-4 py-3 border-b" style={{ borderColor: "#1e1f22" }}>
           <Search className="w-5 h-5" style={{ color: "#949ba4" }} />
-          <input ref={inputRef} type="text" value={query} onChange={handleInput} placeholder="Search messages, tasks, and docs…" className="flex-1 bg-transparent text-white text-sm focus:outline-none" />
+          <input ref={inputRef} type="text" value={query} onChange={handleInput} placeholder="Search… try from:<userId> has:link has:image before:2026-01-01" className="flex-1 bg-transparent text-white text-sm focus:outline-none" />
           {loading && <Loader2 className="w-4 h-4 animate-spin" style={{ color: "#949ba4" }} />}
           <button onClick={onClose}><X className="w-5 h-5" /></button>
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)